tests/testthat/test-Data-class.R

```diff
@@ -1,31 +1,83 @@
-# GeneralData ----
+# GeneralData-class ----
 
-test_that(".GeneralData executes without error for valid object", {
+test_that("GeneralData default constructor .GeneralData
+          executes without error for valid object", {
```

Comment:
if you break the title string please use `paste()` otherwise we have all this whitespace and new line in the string.